src/main/java/com/khadri/jdbc/prepare/data/employee/dao/EmployeeInsertDao.java

```diff
@@ -0,0 +1,34 @@
+package com.khadri.jdbc.prepare.data.employee.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Scanner;
+
+import com.khadri.jdbc.prepare.data.employee.dao.model.Employee;
+
+public class EmployeeInsertDao {
+	public static void insertData(Employee emp) throws Exception {
+		Scanner sc = new Scanner(System.in);
+		Connection con = null;
+		PreparedStatement pstmt = null;
+
+		try {
+			Class.forName("com.mysql.cj.jdbc.Driver");
```

Comment:
write separate class for getting the connection ??.
call the method and use that connection object here.